Packs/SilentPush/Integrations/SilentPush/SilentPush_test.py

```diff
@@ -830,74 +836,77 @@ def test_get_domain_certificates_command_job_status(mock_client, mocker):
 
 
 def test_screenshot_url_command_success(mock_client, mocker):
-    # Mock arguments
     args = {"url": "https://example.com"}
 
-    # Mock response from client
-    mock_response = {"screenshot_url": "https://example.com/screenshot.jpg", "status_code": 200}
+    # Mock the client response
+    mock_response = {"screenshot_url": "https://storage.com/path/screenshot.jpg", "status_code": 200}
     mock_client.screenshot_url.return_value = mock_response
 
-    # Mock requests.get
-    mock_image_response = mocker.Mock(spec=Response)
+    # Properly patch where the function is used, not defined
+    mock_image_response = mocker.Mock()
     mock_image_response.status_code = 200
     mock_image_response.content = b"image content"
-    mocker.patch("requests.Session.request", return_value=mock_image_response)
+    mocker.patch("SilentPush.generic_http_request", return_value=mock_image_response)
 
-    # Mock fileResult
-    mocker.patch("SilentPush.fileResult", return_value={"Type": 3, "FileID": "123", "File": "example_screenshot.jpg"})
+    mock_file_result = mocker.patch("SilentPush.fileResult", return_value={"Type": 3, "FileID": "123"})
+    mock_return_results = mocker.patch("SilentPush.return_results")
 
-    # Call the function
+    # Run command
     result = screenshot_url_command(mock_client, args)
 
-    # Assertions
+    # Validate
     assert isinstance(result, CommandResults)
-    assert result.outputs_prefix == "SilentPush.Screenshot"
-    assert result.outputs_key_field == "url"
     assert result.outputs["url"] == "https://example.com"
     assert result.outputs["status"] == "success"
-    assert result.outputs["screenshot_url"] == "https://example.com/screenshot.jpg"
-    assert "Screenshot captured for https://example.com" in result.readable_output
+    assert result.outputs["screenshot_url"] == mock_response["screenshot_url"]
+    assert result.outputs["file_name"] == "example.com_screenshot.jpg"
```

Comment:
This should throw error, as url is changed to storage.com, same with the next line